packages/itwin/reports-config-widget/src/widget/components/ExtractionStates/SucceededExtractionState.tsx

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { SvgStatusSuccess } from "@itwin/itwinui-icons-color-react";
+import { ReportsConfigWidget } from "../../../ReportsConfigWidget";
+import { ANIMATION_DELAY, ANIMATION_DURATION } from "../Constants";
+
+export const SucceededExtractionState = (animation: string, onAnimationEnd: () => void) => {
```

Comment:
React component should accept a single props argument.

Reply:
`ExtractionStateProps` interface added